Dockerfile.builder

```diff
@@ -15,11 +15,9 @@ RUN yum --enablerepo=centosplus install -y \
       which \
     && yum clean all
 
-# Get glide for Go package management
-RUN cd /tmp \
-    && wget https://github.com/Masterminds/glide/releases/download/v0.12.3/glide-v0.12.3-linux-amd64.tar.gz \
-    && tar xvzf glide-v*.tar.gz \
-    && mv linux-amd64/glide /usr/bin \
-    && rm -rfv glide-v* linux-amd64
+# Get dep for Go package management
+RUN mkdir -p /tmp/go/bin
+ENV GOPATH /tmp/go
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && mv /tmp/go/bin/dep /usr/bin
```

Comment:
go get .. doesn't work here?

Reply:
@aslakknutsen well, I followed the instructions on https://golang.github.io/dep/docs/installation.html

Reply:
@xcoulon Do you follow everything you read on the internet?  ;)